modules/virt-customizing-storage-profile-snapshot-class-web.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * virt/storage/virt-configuring-storage-profile.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-customizing-storage-profile-snapshot-class_web_{context}"]
+= Specifying a volume snapshot class by using the web console
+
+If you are creating a snapshot of a VM, a warning appears if the storage class of the disk has more than one volume snapshot class associated with it. In this case, you must specify one volume snapshot class; otherwise, any disk that has more than one volume snapshot class is excluded from the snapshots list.
+
+You can specify the default volume snapshot class from the {product-title} web console.
+
+.Procedure
+
+. Navigate to *Virtualization* -> *Storage*.
+. Click *VolumeSnapshotClasses*.
```

Comment:
I dont think this step is needed if you go with the above edit.

Reply:
See previous comment about virtualization console navigation.